recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -202,6 +209,50 @@ def __del__(self):
             # created in "/tmp".
             self.driver.quit()
 
+    def startChrome(self, width, height, env):
+        """
+        Starts a Chrome instance. Will use Chromium is Google Chrome is not installed.
+        :param width: the width of the browser window.
+        :param height: the height of the browser window.
+        :param env: the environment variables, including the display to start
+                    the browser in.
+        """
+
+        options = ChromeOptions()
+
+        # "webSocketUrl" is needed for BiDi; this should be set already by
+        # default, but just in case.
+        options.set_capability('webSocketUrl', True)
+
+        options.add_argument('--use-fake-ui-for-media-stream')
+
+        # Allow to play media without user interaction.
+        options.add_argument('--autoplay-policy=no-user-gesture-required')
+
+        options.add_argument('--kiosk')
+        options.add_argument(f'--window-size={width},{height}')
+        options.add_argument('--disable-infobars')
```

Comment:
It seems that `--disable-infobars` is no longer supported by Chromium since a few years ago, so it could be removed. Or did they introduce it again in recent version (although [it does not seem so](https://peter.sh/experiments/chromium-command-line-switches/))?

Reply:
It has been a comes and goes situation. It was removed and added again a couple times if I remember correctly.

Left it in there to avoid adding it again shall the Chromium devs change their mind once more.

Please let me know if it should be removed anyways.

Reply:
Ok, then let's leave it :-)